33-room-occupancy-monitor/README.md

```diff
@@ -87,20 +85,14 @@ Now, to test the PIR motion sensor, simply wave your hand in front of the plasti
 }
 ```
 
-`count` indicates the number of motion events since the last note, while `total` is the total number of events since the device first started running. Similar to the door state notes, the "File" for these notes is `ID#motion.qo`, where ID is that same alphanumeric string that uniquely identifies the reference node.
+`count` indicates the number of motion events since the last note, while `total` is the total number of events since the device first started running. Similar to the door state notes, the "File" for these notes is `motion.qo`, where ID is that same alphanumeric string that uniquely identifies the reference node.
```

Comment:
```suggestion
`count` indicates the number of motion events since the last Note, while `total` is the total number of events since the device first started running. Similar to the door-state Notes, the "File" for these Notes is `motion.qo`, where ID is that same alphanumeric string that uniquely identifies the reference node.
```

Reply:
> where ID is that same alphanumeric string that uniquely identifies the reference node

This isn’t still a thing, right?